app/controllers/admin/home_controller.rb

```diff
@@ -7,12 +7,20 @@ def index
     @activities = Activity.where("start_date >= ?", Date.to_date(Date.today.study_year)).count
 
     @transactions = CheckoutTransaction.count(:all)
+
     @recent = CheckoutTransaction.where("created_at >= ?", Time.zone.now.beginning_of_day).order(created_at: :desc).take(12)
+    @recentparticipants = Payment.where("created_at >= ?", Time.zone.now.beginning_of_day).where(transaction_type: :activity).order(created_at: :desc).take(12)
 
-    @unpayed = Participant.where(:reservist => false, :paid => false).joins(:activity).where('activities.start_date < NOW()').sum(:price) + Participant.where(:reservist => false, :paid => false, :price => nil).joins(:activity).where('activities.start_date < NOW() AND activities.price IS NOT NULL').sum('activities.price')
+    @unpayed = Participant.where(:reservist => false, :paid => false, :price => nil)
+                          .joins(:activity)
+                          .where('activities.is_payable IS TRUE AND activities.price IS NOT NULL')
```

Comment:
Isn't there a ruby method for AND and NOT NULL?

Reply:
This would change the query to a much uglier query afaik there is no real way to do this cleanly